### PR TITLE
[release-4.12] OCPBUGS-21837: nto: pao avoid timeout when there are too many CSV

### PR DESCRIPTION
**Pull request by jlojosnegros**
This is a manual cherry-pick of https://github.com/openshift/cluster-node-tuning-operator/pull/818 

supersedes #830

### CODE REVIEW

**Comment by yanirq** [comment]
/payload-job periodic-ci-openshift-release-master-ci-4.12-upgrade-from-stable-4.11-e2e-gcp-ovn-upgrade

File: test/e2e/basic/csvs.go [hunk 1]
```diff
@@ -30,35 +30,43 @@ const (
...
 	var baseOptions []client.ListOption
-	BeforeAll(func() {
-		var err error
+	var count int
 
-		By("Getting new client")
-		cli, err = newClient()
-		Expect(err).To(Succeed())
+	BeforeEach(func() {
+		if count == 0 {
```
Comment (ffromani) [inline test/e2e/basic/csvs.go:41]: this works iff the test are strictly serial, are them guaranteed serial? otherwise it seems racy (or wrong)

**Review by ffromani — COMMENTED** [review]
LGTM except for part of the ginkgo v1 adaptation

File: test/e2e/basic/csvs.go (hunk 1 above)
Comment (jlojosnegros) [reply test/e2e/basic/csvs.go]: your are totally right.
Thanks for the catch. 

I can't find a way to run the test sequentially using ginkgo v1, so I am thinking about creating the `client` in `BeforeSuite` and let the namespace be created/destroyed on `BeforeEach/AfterEach`.
I am gonna "merge" the `It` clauses so the code is only executed once to save execution time.

**Pull request merged**